.github/workflows/links.yml

```diff
@@ -0,0 +1,57 @@
+name: Links
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 15 * * 1'  # Run every Monday at 3:00 PM UTC
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository  
+        uses: actions/checkout@v4
+
+      - name: Submodule
+        run: git submodule update --init --recursive
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Serve App Locally
+        run: yarn run dev &
+
+      - name: Wait for App to Start
+        run: sleep 20
+      
+      # This will restore the lychee cache
+      - name: Restore lychee cache
+        uses: actions/cache@v4
+        with:
+          path: .lycheecache
+          key: cache-lychee-${{ github.sha }}
+          restore-keys: cache-lychee-
+
+      # This will run the link checker on all markdown files in the pages directory      
+      - name: Link Checker
```

Comment:
Could it be possible to have this workflow for links cleaning but also have another one complete the work done in https://github.com/json-schema-org/website/pull/297 to validate external links in PRs?

Reply:
Could you please explain a bit more what you mean by "link cleaning"? 
Yes, it is possible to check links in the PR. I have already set up the workflow to run manually using the `workflow_dispatch` event. 
Just one question: why check link validation on every PR? Can't we simply check link validation on specific PRs that are related to link changes, and manually trigger the workflow for those?

Thank you : )